components/Boards/PlayerVsPlayer.js

```diff
@@ -5,14 +5,19 @@ import MoveInfo from "../MoveInfo";
 import PlayerNameDisplay from "../PlayerNameDisplay";
 import GameTerminal from "../GameTerminal";
 
-export default function RandomMoveEngine() {
+export default function PlayerVsPlayer() {
   const [game, setGame] = useState(null);
-  const [fen, setFen] = useState(game?.fen());
-  const [previousMove, setPreviousMove] = useState(null);
   const [moveStatus, setMoveStatus] = useState({});
-  const [history, setHistory] = useState([]);
 
-  const latestHistory = history[history.length - 1];
+  //   const [latestResult, setPreviousMove] = useState(null);
```

Comment:
Do we still need this? If not, i don't think we want it making it into `main`.

Reply:
Ups, that one got overlooked...thanks.